crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -82,16 +84,38 @@ impl ModuleLoader {
       plugin_driver,
     });
 
+    let mut intermediate_normal_modules = IntermediateNormalModules::new();
+    let mut symbols = Symbols::default();
+    let runtime_id = intermediate_normal_modules.alloc_module_id(&mut symbols);
+
+    let task = RuntimeNormalModuleTask::new(runtime_id, tx_to_runtime_module);
+    let handle = tokio::runtime::Handle::current();
+
+    #[cfg(target_family = "wasm")]
+    {
+      // could not block_on/spawn the main thread in WASI
```

Comment:
Changes are fine. But this comment seems a little confused with `could not block_on` and the following `handle.block_on(`.